### ISSUE TITLE
(Feature Request) Automatic related sync

### ISSUE THREAD

**Issue by shedman214** [Snd-R/komf#56]
A feature where it automatically matches how some manga are related 
![image](https://user-images.githubusercontent.com/32603273/233713897-3a8b7f1b-fcf0-4780-9ae9-35038099ad38.png)
![image](https://user-images.githubusercontent.com/32603273/233713921-cc57ffb9-8ca4-451d-bf41-9730649c6557.png)

**Comment by garvongithub** [comment]
Yes this can be a nice feature, it can create collections based on realease date.

**Comment by ElryWeeb** [comment]
That would be extremly taxing on the Server to do that with every scanned series and would increase the time komf needs per series significantly.
As it needs to start searching if you even have the related series in you library.
Im strongly against this or that its definetly disabled by default.
